t/shared.t

```diff
@@ -33,5 +32,23 @@ is( $hash{multi}->{key1},      "foo", "multidimension, key1" );
 is( $hash{multi}->{arr1}->[0], "bar", "multidimension, arr1 - key0" );
 is( $hash{multi}->{arr1}->[1], "baz", "multidimension, arr1 - key1" );
 
-unlink("vars.db");
-unlink("vars.db.lock");
+{
+    @array = (0);
+    my @pids;
+    for my $i (0..5) {
+        my $pid = fork();
+        if ($pid == 0) {
+            # child
+            sleep 1;
```

Comment:
That still hangs on my machine.  

I don't think its a reliable test with `pause()`.  There is a race condition.  The parent will send the 'CONT' signal as soon as it can.  And the child processes need to fork and setup the signal handler before that happens.

To make this work reliably:
1. the parent should pause before sending the 'CONT' signal
2. the parent should wake up and send a 'CONT' signal each time a child process is ready and has its signal handler setup.  

This doable, but I don't think the complexity is worth it.